config

```diff
@@ -13,8 +13,8 @@ prefix = labs
 
 # CCID list
 # Provide no value to collect all repos
-list = ccid_list.txt
+ccid_list = ccid_list.txt
 
 # Deadline of the given assignment or lab in format \"YYYY-MM-DD HH:MM:SS -0700\" where -0700 indicates the MST time zone
 # Provide no value to collect repos with the latest commit
-deadline = 2021-01-13 00:00:00 -700
\ No newline at end of file
+deadline = 2021-01-13 00:00:00 -0700
```

Comment:
perhaps that missing 0 was the issue?

Reply:
The comment above this line was correct, I'm not sure if he added 0.